agent/engine/docker_task_engine_test.go

```diff
@@ -744,6 +746,95 @@ func TestCreateContainerMergesLabels(t *testing.T) {
 	taskEngine.(*DockerTaskEngine).createContainer(testTask, testTask.Containers[0])
 }
 
+// TestCreateContainerV3EndpointIDGenerateFailure tests that when we successfully generate
+// v3 endpoint id for container, we will inject v3 endpoint to it
+func TestCreateContainerV3EndpointIDGenerateSucceed(t *testing.T) {
```

Comment:
Is it possible to put `TestCreateContainerV3EndpointIDGenerateSucceed` and `TestCreateContainerV3EndpointIDGenerateFailure` in the same function? They share the most of the logic.